docs/yaml/04-service.yaml

```diff
@@ -0,0 +1,15 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: gangway
```

Comment:
I think this may be problematic as it will expose `GANGWAY_PORT` for environment-based service discovery.  This also happens to be an environment variable exposed by envconfig. This would be true for any Service named `gangway`.

Reply:
Hmm -- you have to pick *some* port here.  I'm going to switch this all to use port names (and use `http`) vs using an explicit port number.  Also going to have the service expose 80.